src/python/bot/testcase_manager.py

```diff
@@ -577,6 +577,12 @@ def __init__(self,
     else:
       engine_impl = None
 
+    # TODO(mbarbella): Remove this check once AFL is fully converted to the new
+    # engine interface.
+    if (fuzz_target and fuzz_target.engine == 'afl' and
+        not environment.get_value('USE_AFL_ENGINE')):
+      engine_impl = None
```

Comment:
Can just put this in engine.get() instead of duplicating the checks in two places.

Reply:
Good call. Done.